c10/mobile/CPUCachingAllocator.h

```diff
@@ -10,6 +10,39 @@
 #include <c10/util/SmallVector.h>
 #include <c10/util/flat_hash_map.h>
 
+/*
+ * CPUCachingAllocator:
```

Comment:
nit: can you put a brief summary of the disclaimer first, like:

`DISCLAIMER: this is subject to change (beta) and only supported on mobile builds.  See below for more information"